src/Teste.java

```diff
@@ -8,6 +9,11 @@ public static void main(String[] args) {
 		ClasseExemplo c1 = new ClasseExemplo();
 		
 		c1.funcaoAlgumaCoisa();
+		
```

Comment:
Po, Gabriel. Acho que ficou falando um system.out.println pra pedir para o usuario colocar o nome
```suggestion
		System.out.print("Insira o nome do usuário: ");
```

Reply:
faltou mesmo, e tambem o sc.close();

Reply:
Boa! Mas acho que pode deixar assim mesmo já que é exemplo